examples/cron-when-limit-concurrency.yaml

```diff
@@ -0,0 +1,21 @@
+# Example of how a "when" clause can be used
+# to control concurrency
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: sleep-when-limit-concurrency
+spec:
+  schedule: "* * * * *"
+  timezone: "America/Los_Angeles"   # Default to local machine timezone
+  startingDeadlineSeconds: 0
+  suspend: false                    # Set to "true" to suspend scheduling
```

Comment:
the above 3 fields are not relevant to this CronWorkflow, so should be removed. Otherwise they show up in other places in the Fields Reference and can be confusing as an example